job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/QueueServices.java

```diff
@@ -120,22 +86,24 @@ public void sendMessage(
                 getStatusCheckIntervalMillis(ScheduledExecutorConfig.getStatusCheckIntervalSeconds()),
                 statusCheckUrl, ScheduledExecutorConfig.getTrackingPipe(), workerAction.getTargetPipe());
 
-        final TaskMessage taskMessage = new TaskMessage(
+        LOG.debug("taskdata boko {}", taskDataObj);
+        final QueueTaskMessage queueTaskMessage = new QueueTaskMessage(
                 taskId,
                 workerAction.getTaskClassifier(),
                 workerAction.getTaskApiVersion(),
-                taskData,
+                taskDataObj,
                 TaskStatus.NEW_TASK,
-                Collections.<String, byte[]>emptyMap(),
+                Collections.emptyMap(),
```

Comment:
Is this change required?

Reply:
I think that's a good change given that we don't need to specify the type and because the map is empty, but I understand that it's more some tidying up rather that being part of the current jira, so I restored